h2o-algos/src/main/java/hex/glm/GLM.java

```diff
@@ -1357,6 +1357,7 @@ public void computeImpl() {
     }
 
     @Override public boolean onExceptionalCompletion(Throwable t, CountedCompleter caller){
+      super.onExceptionalCompletion(t, caller);
       doCleanup();
       return true;
```

Comment:
Should it not return result from call to `super`?

Reply:
No.